spec/lib/msf/core/modules/metadata/search_spec.rb

```diff
@@ -72,65 +107,63 @@
     end
 
     context 'on a module with the author "joev"' do
-      let(:opts) { ({ 'Author' => ['joev'] }) }
+      let(:opts) { ({ 'author' => ['joev'] }) }
```

Comment:
Previously these opts were used to hydrate a module, now they're hydrating the search cache - as a result I've had to update some of the keys.